src/templates/base/base.html

```diff
@@ -611,6 +616,16 @@
             }
           }
         });
+
+        var ebookLink = document.querySelector('#ebook-toc-download');
+        ebookLink && ebookLink.addEventListener("click", function (event) {
+          gtag('event', 'ebook-click', { 'event_category': 'clicks', 'event_label': 'toc-page', 'transport_type': 'beacon', 'value': 1 })
```

Comment:
How useful is it for the `event_label` to tell us on what page the event happened? Doesn't GA already know that?

Reply:
Not sure it’s that useful at all tbh. But previous event had `toc-page` so had to replace it. Happy to go with just `toc-menu`.